reporter/Report.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
```

Comment:
I don't think we need to check in Report.html. This can be removed from the PR

Reply:
This is a custom report i am generating for accessibility so i have to pass this html template for report generation